readme.md

```diff
@@ -16,19 +16,30 @@ $ source .venv/bin/activate
 (.venv) $ pip3 install -r requirements.txt
 ```
 
-Note: To step out of the virtual environment, run `$ deactivate`.
+If you will finish working in virtual environment you can step out of it:
```

Comment:
s/If you will finish working in virtual environment/When finished working inside the virtual environment

Reply:
Fixed